src/main/java/it/gov/pagopa/payhub/activities/connector/debtposition/DebtPositionTypeOrgServiceImpl.java

```diff
@@ -42,4 +42,15 @@ public IONotificationDTO getDefaultIONotificationDetails(Long debtPositionTypeOr
             return null;
         }
     }
+
+    @Override
+    public DebtPositionTypeOrg getDebtPositionTypeOrgByInstallmentId(Long installmentId) {
+        log.info("Fetching DebtPositionTypeOrg with installmentId: {}", installmentId);
+        try {
```

Comment:
here I expect you have nothing to catch...
why should you transform a generic exception into null?